sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -44,6 +44,7 @@
     private static final String CLIENT_TELEMETRY_ENABLED = "COSMOS.CLIENT_TELEMETRY_ENABLED";
     private static final String CLIENT_TELEMETRY_SCHEDULING_IN_SECONDS = "COSMOS.CLIENT_TELEMETRY_SCHEDULING_IN_SECONDS";
     private static final String CLIENT_TELEMETRY_ENDPOINT = "COSMOS.CLIENT_TELEMETRY_ENDPOINT";
+    private static final String ENVIRONMENT_NAME = "COSMOS.ENVIRONMENT_NAME";
```

Comment:
Since this is only for CTL, in my opinion, we should rename this to `CTL_ENVIRONMENT_NAME`.

Reply:
Discussed offline , environment name is not specific to ctl , it can be any workload test in various env.